x-pack/plugins/ml/public/embeddables/anomaly_charts/embeddable_anomaly_charts_container.tsx

```diff
@@ -99,7 +99,7 @@ export const EmbeddableAnomalyChartsContainer: FC<EmbeddableAnomalyChartsContain
   const resizeHandler = useCallback(
     throttle((e: { width: number; height: number }) => {
       if (Math.abs(chartWidth - e.width) > 20) {
-        setChartWidth(e.width);
+        requestAnimationFrame(() => setChartWidth(e.width));
```

Comment:
what are the benefits of using `requestAnimationFrame` here?

Reply:
Thanks for catching that - this is redundant. I have removed it here https://github.com/elastic/kibana/pull/101404/commits/080c6318a6daa3c15bcccf97854173792b21feeb